packages/main/src/backend/export/outputVendors/ynab/ynab.ts

```diff
@@ -196,7 +204,8 @@ export function isSameTransaction(
     Math.abs(transactionToCreate.amount - transactionFromYnab.amount) < 1000 &&
     // In a transfer transaction the payee name changes, but we still consider this the same transaction
     (areStringsEqualIgnoreCaseAndWhitespace(transactionToCreate.payee_name, transactionFromYnab.payee_name) ||
-      isATransferTransaction)
+      isATransferTransaction ||
+      isSameImportId(transactionToCreate, transactionFromYnab))
```

Comment:
This `return` condition becomes very complicated.

Could you break it into variables to make it more readable?

For example, take the `()` part to `sameTransactionDifferentPayeeName = ...`

Reply:
Done